packages/investor-yearn/src/YearnOpportunity.ts

```diff
@@ -176,12 +176,13 @@ export class YearnOpportunity
     wallet: HDWallet
     tx: PreparedTransaction
     feePriority?: FeePriority
+    accountNumber?: number
   }): Promise<string> {
-    const { wallet, tx, feePriority } = input
+    const { accountNumber, wallet, tx, feePriority } = input
     const feeSpeed: FeePriority = feePriority ? feePriority : 'fast'
     const chainAdapter = this.#internals.chainAdapter
 
-    const path = toPath(chainAdapter.buildBIP44Params({ accountNumber: 0 }))
+    const path = toPath(chainAdapter.buildBIP44Params({ accountNumber: accountNumber ?? 0 }))
```

Comment:
we cannot safely fall back here - we should throw if not provided

Reply:
That's what I was thinking as well, ideally would prefer for the type-system to catch this so we never have to add runtime logic for this, but this would do the trick for now.
Are we happy with this *actually temporary runtime workaround* as a type-safety substitute?

Reply:
Added and rebased interactive to ease landing the `BREAKING CHANGE:` commit body:

https://github.com/shapeshift/lib/blob/474a74421cde7db823dbd0d4b74c9e6affb7e42a/packages/investor-yearn/src/YearnOpportunity.ts#L184